### ISSUE TITLE
[Feature Request] Add button to open scene in VLC from grid view.

### ISSUE THREAD

**Issue by zezimarunescape999** [cooperdk/YAPO-e-plus#50]
It would be nice to be able to launch a scene in VLC from grid view without having to first open the page for the scene. Perhaps a VLC button could be overlaid in the corner of the thumbnail on hover.

**Comment by cooperdk** [comment]
I agree this is a good option if you use VLC to watch videos and not the embedded player. I'll add it to the tasks.
Thanks for your request and time.

**Comment by zezimarunescape999** [comment]
> I agree this is a good option if you use VLC to watch videos and not the embedded player. I'll add it to the tasks.
> Thanks for your request and time.

Thanks! I use SVP ([SmoothVideo Project](https://www.svp-team.com/)) which interpolates low framerate videos to 60 FPS or higher. It runs in VLC (and other players) so I always prefer to watch my videos with VLC when possible.

**Comment by cooperdk** [comment]
OK. I don't remember if I actually turned off the ability to use VLC on scenes that can play in the browser. This could be a toggleable option in settings.
